Example/Backpack/SwiftUI/Components/Calendar/CalendarExampleRangeView.swift

```diff
@@ -22,13 +22,15 @@ import Backpack_SwiftUI
 
 struct CalendarExampleRangeView: View {
     @State var selection: CalendarRangeSelectionState?
-    
+    private var monthScroll: MonthScroll?
+
     let validRange: ClosedRange<Date>
     let calendar: Calendar
     let formatter: DateFormatter
     let showAccessoryViews: Bool
-    
-    init(showAccessoryViews: Bool) {
+    let makeInitialMonthScroll: Bool
```

Comment:
why is this property? seems only used within the init, should be removed i think